scripts/fast-reboot

```diff
@@ -70,44 +88,83 @@ docker exec -i bgp killall -9 bgpd
 docker kill lldp > /dev/null
 
 # Kill teamd, otherwise it gets down all LAGs
+# Note: teamd must be killed before syncd, because it will send the last packet through CPU port
+# TODO: stop teamd gracefully to allow teamd to send last valid update to be sure we'll have 90 seconds reboot time
 docker kill teamd > /dev/null
 
+# Freeze orchagent for warm restart
+if [[ "$REBOOT_TYPE" = "warm-reboot" ]]; then
+    docker exec -i swss /usr/bin/orchagent_restart_check || {
+        echo "RESTARTCHECK failed\n" >&2
+        # TODO: remove this sleep if we have better timing control
+        sleep 1
+        exit 10
+    }
+fi
+# Set whole system warm reboot flag
+config warm_restart enable system
```

Comment:
For fast reboot, should this step be skipped?

Reply:
Thanks! Fixed.

---
In reply to: [225346129](https://github.com/Azure/sonic-utilities/pull/338#discussion_r225346129) [](ancestors = 225346129)